### PR TITLE
Update custom_instances toggle desc

### PR DESCRIPTION
**Pull request by wpride**
Enikshay is dead, long live `CUSTOM_INSTANCES`

### CODE REVIEW

File: corehq/toggles.py [hunk 1]
```diff
@@ -860,8 +860,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
...
 CUSTOM_INSTANCES = StaticToggle(
     'custom_instances',
-    'Enikshay: Inject custom instance declarations',
+    'Inject custom instance declarations',
     TAG_CUSTOM,
+    description=(
+        'Enables the insertion of custom instances into a case list configuration. '
+        'Currently used by SimPrints-integrated projects.'
+    ),
     namespaces=[NAMESPACE_USER, NAMESPACE_DOMAIN],
```
Comment (emord) [inline corehq/toggles.py]: Do any users have this enabled? I think we'll only want to have this enabled on the domain to make sure that app edits don't wipe this value? (I realize that this was pre-existing, but it does seem bad)

**Review by emord — APPROVED** [review]

File: corehq/toggles.py (hunk 1 above)
Comment (wpride) [reply corehq/toggles.py]: Good call. No users on `www` or `india`- think that covers it?

Comment (emord) [reply corehq/toggles.py]: Yeah. I think that should cover it

**Review by emord — APPROVED** [review]

File: corehq/toggles.py [hunk 2]
```diff
@@ -860,9 +860,13 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 CUSTOM_INSTANCES = StaticToggle(
     'custom_instances',
-    'Enikshay: Inject custom instance declarations',
+    'Inject custom instance declarations',
     TAG_CUSTOM,
```
Comment (emord) [inline corehq/toggles.py]: Should this have been changed to a solutions flag?

**Pull request merged**